lib_test/compat/dune

```diff
@@ -0,0 +1,4 @@
+(tests
+ (libraries mustache ezjsonm)
+ (names user_program)
+ (deps user_program.exe))
```

Comment:
Dune already adds the dependency on the binary automatically.

Reply:
I removed the `(deps ..)` stanza, thanks.